consent-ws/src/test/java/org/broadinstitute/consent/http/AbstractTest.java

```diff
@@ -180,4 +180,33 @@ public void mockValidateTokenResponse() throws IOException {
         Mockito.when(entity.getContent()).thenReturn(inputStream);
         OAuthAuthenticator.getInstance().setHttpClient(client);
     }
+
+    Consent generateNewConsent(UseRestriction useRestriction, DataUseDTO dataUse) {
+        Timestamp createDate = new Timestamp(new Date().getTime());
+        return new ConsentBuilder().
+                setRequiresManualReview(false).
+                setUseRestriction(useRestriction).
+                setDataUse(dataUse).
+                setName(UUID.randomUUID().toString()).
+                setCreateDate(createDate).
+                setLastUpdate(createDate).
+                setSortDate(createDate).
+                build();
+    }
+
+    protected MongoClient setUpMongoClient() throws IOException {
```

Comment:
Three test classes were doing the same thing in copy/pasta mode. Abstracted here to clean up the duplicate code and to prep for upgrading flapdoodle (see https://broadinstitute.atlassian.net/browse/GAWB-2440).